plasmapy/formulary/quantum.py

```diff
@@ -36,6 +36,19 @@ def deBroglie_wavelength(V: u.m / u.s, particle) -> u.m:
     Calculates the de Broglie wavelength.
 
     **Aliases:** `lambdaDB_`
+    
+    Formula
+    -------
+    The de Broglie wavelength is given by
+
+    .. math::
+
+        \lambda_{dB} = \frac{h}{p} = \frac{h}{\gamma m V}
+
+    where :math:`h` is the Planck constant, :math:`p` is the
+    relativistic momentum of the particle, :math:`gamma` is the
+    Lorentz factor, :math:`m` is the particle's mass, and :math:`V` is the
+    particle's velocity.
```

Comment:
Delete the "Formula" section header and move the "equation" block between the one-line description and "Aliases".

@namurphy and @StanczakDominik, What do you think about moving the equation up to the top of the docstring?  I prefer it because I think it's a nuisance for the user to have to scroll down the documentation to find the equation associated with the formula.

Reply:
I don't mind either way, to be honest!